organizers/2017-07-18.html

```diff
@@ -0,0 +1,86 @@
+---
+layout: organizers
+---
+
+<section>
+    <h1>The London Django Meetup Group</h1>
+    <h2>18th July 2017</h2>
+</section>
+
+<section data-background='#eee'>
+    <h1>Sponsorship</h1>
+    <p>Tonight's meetup is kindly sponsored by:</p>
+    <ul>
+        <li><strong>Festicket</strong></li>
+        <li><strong>Time Out</strong></li>
+        <li><strong>Funding Options</strong></li>
+    </ul>
+</section>
+
+<section data-background='#eee'>
+    <object type="image/svg+xml" data="/img/festicket.svg" style="width:40%">Festicket</object>
+    <p>
+        <em>“Festival package travel startup. 50+ staff, high pay, ping pong + beer.”</em>
+    </p>
+</section>
+
+<section data-background='#eee'>
+    <img data-src="/img/timeout.png" style="width:40%" alt="Time Out" class="plain"></img>
+    <p>
+        <em>The trusted global platform that inspires and enables people to experience the best of the city. Operating in 107 cities, across 39 countries, it has an average monthly global audience reach of over 111 million across all platforms.</em>
+    </p>
+</section>
+
+<section data-background='#eee'>
+    <img data-src="/img/funding-options-logo-black.svg" style="width:40%" alt="Funding Options" class="plain"></img>
+    <p>
+        <em>Funding Options is the free marketplace for business finance, using matching technology to help UK businesses access tens
+        of millions of pounds in finance every year.
+        </em>
+    </p>
+</section>
+
+<section style="text-align: left">
+    <h2 style="text-align: center;">Today's Schedule - Talks</h2>
+    <ol>
+        <li style="overflow: hidden;">
+            <img src="https://image.ibb.co/k9QgfF/yw.jpg"
```

Comment:
Hmm - it's not what we've done before.  This is happening because CodeNode haven't uploaded the images, which they usually have done by now.